### PR TITLE
*: fix typos reported by Hunspell

### LINKED ISSUE #5945: git-show-branch: bug
- Keep going a given number of commits past the common ancester:

/tldr/pages/common/git-show-branch:34

Spelling error

### PR DESCRIPTION
**Pull request by bl-ue**
Fixes a manually corrected selection of typos reported by [Hunspell](https://hunspell.github.io/) in all of the English pages.

Closes #5945

### CODE REVIEW

**Review by einverne — APPROVED** [review]
Cool, I am wondering should we add a glossary to CI to warn the example like Javascript -> JavaScript?

**Comment by bl-ue** [comment]
Well, as a matter of fact, I'm thinking of discussing integrating [codespell](https://github.com/codespell-project/codespell) into GitHub Actions, and I'm going to try to add `Javascript` → `JavaScript` to their dictionary (as I'm going to do with all of these genuine misspellings).  `codespell` would then error and prevent a merge if it found some typos.  The problem is, they [don't support multiple languages](https://github.com/codespell-project/codespell/issues/1435).  I'll experiment with Hunspell on different languages later.

File: pages.de/common/npm.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # npm
 
-> Ein Kommandozeilenwerkzeug für die Verwaltung von Javascript und Node.js Paketen (Packages).
+> Ein Kommandozeilenwerkzeug für die Verwaltung von JavaScript und Node.js Paketen (Packages).
```
Comment (sbrl) [inline pages.de/common/npm.md:3]: Weird. I'd personally always write it `Javascript`, but apparently https://js.org/ agrees with you in this change :woman_shrugging:

File: pages/osx/uuidgen.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # uuidgen
 
-> Generate new UUID (Universally Unique IDentifier) strings.
+> Generate new UUID (Universally Unique Identifier) strings.
```
Comment (sbrl) [inline pages/osx/uuidgen.md:3]: This should not be changed. The whole point of the extra capitalisation here is to reference the acronym `UUID` here.

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the edits! I've left some comments below for you to review.

File: pages.de/common/npm.md (hunk 1 above)
Comment (bl-ue) [reply pages.de/common/npm.md:3]: Yeah.  `Javascript` is definitely incorrect.  `JavaScript` is the official standard spelling.

File: pages/osx/uuidgen.md (hunk 2 above)
Comment (bl-ue) [reply pages/osx/uuidgen.md:3]: No problem.  I didn't think it was a good idea to change.

Comment (bl-ue) [reply pages/osx/uuidgen.md:3]:
```suggestion
> Generate new UUID (Universally Unique IDentifier) strings.
```

File: pages.zh/osx/port.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # port
 
-> MacOS 包管理器软件（类似 brew).
+> macOS 包管理器软件（类似 brew).
```
Comment (navarroaxel) [inline pages.zh/osx/port.md:3]: What if this is the star of the sentence?

File: pages/common/meson.md [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 `meson init --language={{c}} --name={{myproject}} --version={{0.1}}`
 
-- Configure builddir with default values:
+- Configure build directory with default values:
```
Comment (navarroaxel) [inline pages/common/meson.md:10]: `builddir` looks like a domain specific language https://mesonbuild.com/Builtin-options.html#base-options :thinking:

File: pages.zh/osx/port.md (hunk 3 above)
Comment (bl-ue) [reply pages.zh/osx/port.md:3]: I'm not exactly sure what you mean.  The [English page](https://github.com/tldr-pages/tldr/blob/6a058b56fe090debff720d06196c02199ea5644c/pages/osx/port.md) says this:

> Package manager for macOS.

Google translates that Chinese to `macOS package manager software (similar to brew).`

The term "MacOS" isn't correct anymore.  Apple has replaced it with "macOS" (to my knowledge).

File: pages/common/meson.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/meson.md:10]: I'll fix it.

Comment (bl-ue) [reply pages/common/meson.md:10]: What do you think about this?
```suggestion
- Configure the builddir with default values:
```

File: pages.zh/osx/port.md (hunk 3 above)
Comment (navarroaxel) [reply pages.zh/osx/port.md:3]: I mean a sentence should start with an upper case letter. But maybe I'm wrong

File: pages/common/meson.md (hunk 4 above)
Comment (navarroaxel) [reply pages/common/meson.md:10]: Looks good.

File: pages.zh/osx/port.md (hunk 3 above)
Comment (marchersimon) [reply pages.zh/osx/port.md:3]: I don't think that applies to things like product names. On its [wikipedia article](https://en.wikipedia.org/wiki/MacOS) it's also lowercase, even at the start of the sentence.

File: pages/common/aria2.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # aria2
 
-> A lightweight multi-protocol & multi-source command-line download utility.
+> A lightweight multi-protocol & multi-source command line download utility.
```
Comment (nicokosi) [inline pages/common/aria2.md:3]: I thought `command-line` was OK. The official git documentation uses both `command line` and `command-line` ([example](https://git-scm.com/docs/git)). Any references? :innocent:

File: pages/common/fdroid.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # fdroid
 
 > F-Droid build tool.
-> F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform.
+> F-Droid is an installable catalog of FOSS (Free and Open Source Software) applications for the Android platform.
```
Comment (nicokosi) [inline pages/common/fdroid.md:4]: Should US English be preferred over UK English? :innocent: 
https://www.wordreference.com/enfr/catalogue

File: pages/common/indent.md [hunk 7]
```diff
@@ -11,6 +11,6 @@
 
 `indent --gnu-style {{path/to/source.c}} -o {{path/to/indented_source.c}}`
 
-- Format C/C++ source according to the style of Kernigan & Ritchie (K&R), no tabs, 3 spaces per indent, and wrap lines at 120 characters:
+- Format C/C++ source according to the style of Kernighan & Ritchie (K&R), no tabs, 3 spaces per indent, and wrap lines at 120 characters:
```
Comment (nicokosi) [inline pages/common/indent.md:14]: Nice one. :+1: :sunglasses:

File: pages/common/sonar-scanner.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # sonar-scanner
 
-> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like maven, gradle , etc.
+> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like maven, gradle, etc.
```
Comment (nicokosi) [inline pages/common/sonar-scanner.md:3]:
```suggestion
> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like Maven, Gradle, etc.
```

Comment (nicokosi) [reply pages/common/sonar-scanner.md:3]: Is it me... or this sentence does not make any sense? :innocent:

File: pages/common/fdroid.md (hunk 6 above)
Comment (nicokosi) [reply pages/common/fdroid.md:4]: If not, shouldn't we avoid changing English words from UK to US and vice-versa? (it could be endless :blush:)

Comment (nicokosi) [reply pages/common/fdroid.md:4]: I found the response :innocent: in https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#default-language-for-newly-added-pages (RTFM :wink:):

> The default language used for pages is English (US).

File: pages.zh/osx/port.md (hunk 3 above)
Comment (nicokosi) [reply pages.zh/osx/port.md:3]: Here is the recommendation from [Google developer documentation style guide](https://developers.google.com/style/capitalization):

> If an official name begins with a lowercase letter, then put it in lowercase even at the start of a sentence. But it's better to revise the sentence to avoid putting a lowercase word at the start, if possible.

Comment (nicokosi) [reply pages.zh/osx/port.md:3]: I could find a similar reference in the `The Chicago Manual of Style` and the `Oxford University Style Guide`, but I only spent a few seconds searching... :innocent:

File: pages/common/indent.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/indent.md:14]: Yeah, I particularly enjoyed fixing this one as well ;)

Comment (marchersimon) [reply pages/common/indent.md:14]: > no tabs, 3 spaces per indent

Why would anybody want to use that? 🤣

File: pages/common/sonar-scanner.md (hunk 8 above)
Comment (marchersimon) [reply pages/common/sonar-scanner.md:3]:
```suggestion
> SonarScanner is a generic scanner for SonarQube for projects, which do not use any specific build tool like maven, gradle, etc.
```
Still not happy with the `for SonarQube for projects`, but at least it's a bit better now

File: pages.zh/osx/port.md (hunk 3 above)
Comment (bl-ue) [reply pages.zh/osx/port.md:3]: Then let's keep it.

>>But it's better to revise the sentence to avoid putting a lowercase word at the start, if possible.

Exactly what I would say. 👍🏻

File: pages/common/aria2.md (hunk 5 above)
Comment (bl-ue) [reply pages/common/aria2.md:3]: Personally, I prefer "command-line" (not sure why 🤔).  But it's a tie! 🤞🏻

These stats are taken from the latest `main` @ 5ba22052b74d11926b66444e53014e5c74630fe1:
```
command line: 116
command-line: 113
commandline:  65
```

File: pages/common/indent.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/indent.md:14]: Idk — Google redirects me to "Kernighan" too. 🤷🏻‍♂️

File: pages/common/sonar-scanner.md (hunk 8 above)
Comment (bl-ue) [reply pages/common/sonar-scanner.md:3]: You guys are both totally right — I didn't even pay a speck of attension to the rest of the sentence until _just_ now 😄

File: pages/common/fdroid.md (hunk 6 above)
Comment (sbrl) [reply pages/common/fdroid.md:4]: Hey there, @nicokosi! Generally speaking we prefer to avoid hiding comments unless absolutely necessary, as it obscures the conversation that has taken place. You can always click the "resolve conversation" button instead. As a side note, it's generally good practice not to hit resolve conversation on important conversations that have occurred on a pull request either, as it makes them more obvious when revisiting later.

File: pages/common/sonar-scanner.md (hunk 8 above)
Comment (bl-ue) [reply pages/common/sonar-scanner.md:3]: This is much better, right?
```suggestion
> SonarScanner is a generic scanner for SonarQube projects that do not use build tools such as Maven, Gradle, or Ant.
```

Comment (sbrl) [reply pages/common/sonar-scanner.md:3]: Hrm, this is an awkward one - I completely agree on the double `for` there. What about this?
```suggestion
> SonarScanner is a generic SonarQube scanner for projects that do not use any specific build tool such as Maven or gradle.
```

Ref https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/ for additional docs on the thing I think

File: pages/common/meson.md [hunk 9]
```diff
@@ -7,7 +7,7 @@
 
 `meson init --language={{c}} --name={{myproject}} --version={{0.1}}`
 
-- Configure build directory with default values:
+- Configure the builddir with default values:
```
Comment (sbrl) [inline pages/common/meson.md:10]: This doesn't make sense either. What about this?
```suggestion
- Configure the build directory with default values:
```

...unless `builddir` is a specific technical term in the contact of `meson`?

Comment (bl-ue) [reply pages/common/meson.md:10]: I made this exact change, but @nicokosi pointed at that, in fact, it _is_ specific to meson: https://github.com/tldr-pages/tldr/pull/5848#discussion_r624564008

File: pages/common/fdroid.md (hunk 6 above)
Comment (sbrl) [reply pages/common/fdroid.md:4]: And to answer your question yeah, we do generally use en_US for English pages. But it's not a completely hard and fast rule - there's no need to go through all our existing pages and fix the language. I'm sure there are some notable exceptions.

File: pages/common/indent.md (hunk 7 above)
Comment (sbrl) [reply pages/common/indent.md:14]: I knew I'd heard that name somewhere before. Brian Kernighan co-authored the first book on the C programming language, along with Dennis Ritchie. I remember seeing his name on a book in my Academic Support Tutor's office I think, and also on a picture in a slide deck in a lecture. [According to Wikipedia](https://en.wikipedia.org/wiki/Brian_Kernighan), Brian Kernighan did not have a hand in designing C in the first place though.

File: pages/common/fdroid.md (hunk 6 above)
Comment (nicokosi) [reply pages/common/fdroid.md:4]: No problem for not hiding comments (I did it because I answered my own questions 😉).

File: pages/common/indent.md (hunk 7 above)
Comment (nicokosi) [reply pages/common/indent.md:14]: > > no tabs, 3 spaces per indent
> 
> Why would anybody want to use that? 🤣

I work on a legacy code base that uses 3-space indentation. I think it's a compromise between 2 and 4. 😜

File: pages.zh/osx/port.md (hunk 3 above)
Comment (Waples) [reply pages.zh/osx/port.md:3]: this reminds me of the comments on my `sic` PR, where I got pushed into using uppercase..

File: pages/common/aria2.md (hunk 5 above)
Comment (Waples) [reply pages/common/aria2.md:3]: `commandline` is wrong
I also vote for `command-line`, though if you googe around, in textual context its more referred to as `command line`. But I like `command-line` better.

File: pages/common/sonar-scanner.md (hunk 8 above)
Comment (nicokosi) [reply pages/common/sonar-scanner.md:3]: `gradle`-> `Gradle` 😇

File: pages.de/common/npm.md (hunk 1 above)
Comment (CleanMachine1) [reply pages.de/common/npm.md:3]: if it was Js when shortened it would make sense for it to be Javascript.
JS is definitely correct

**Comment by Waples** [comment]
@bl-ue There seem to be some conflicts you might wanna check out 😉

**Comment by marchersimon** [comment]
@bl-ue you mean because I changed the title and tag? You still edit a few translated pages, so I thought that enough to change it.

**Comment by patricedenis** [comment]
Can someone explain me how to solve the conflicts?
Because when I tried it with the Github Web GUI by clicking the "Resolve conflits" button editing them as asked and marked them as resolved. I got back to this page and saw that the conflicts were not resolved.

**Comment by bl-ue** [comment]
@marchersimon - I assumed that you changed `English pages` to `All pages` because had previously written `Other languages coming soon!` in this PRs description — I had changed my mind, though, but I  forgot to clarify that.

@patricedenis - https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github.  I'll fix the conflicts in this PR for now.

**Comment by patricedenis** [comment]
> @patricedenis - https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github. I'll fix the conflicts in this PR for now.

All right then. I did not see the "commit changes" button so that's why.
Thank you again for your feedback.

**Comment by bl-ue** [comment]
@patricedenis please see https://github.com/tldr-pages/tldr/pull/5848/commits/29d67af0cf5077d9f0227818459d8de985469f89 — do the changes to the French look good, or have you any suggestions regarding them?

**Comment by bl-ue** [comment]
@patricedenis I'm sorry, do you mind adding those as suggestions in the [Files Changed](https://github.com/tldr-pages/tldr/pull/5848/files) tab?  That way I can apply them easily.

File: pages.fr/common/convert.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # convert
 
-> Outil de conversion d'image de Imagemagick.
+> Outil de conversion d'image de ImageMagick.
```
Comment (patricedenis) [inline pages.fr/common/convert.md:3]:
```suggestion
> Outil de conversion d'image d'ImageMagick.
```

**Comment by patricedenis** [comment]
> @patricedenis I'm sorry, do you mind adding those as suggestions in the [Files Changed](https://github.com/tldr-pages/tldr/pull/5848/files) tab? That way I can apply them easily.

Sorry
When I use your link, I can't find the file I edited in the #29d67af
And when I follow the link of the commit I can't find the "Files Changed" Tab or the suggestion button 
image

**Comment by patricedenis** [comment]
When I look this Pull request #5848, I can only see 2 pages.fr files that were committed but when I look into your commit #29d67af commit I can't find the tab "Files Changed" to emit suggestions.

**Comment by patricedenis** [comment]
@bl-ue where should I go to find the correct "Tab" button so I could continue?

**Comment by bl-ue** [comment]
Hmmm, I'm not sure what's going wrong.  I'll just apply those suggestions manually 👍🏻

**Comment by bl-ue** [comment]
GH messed up my URL — I meant you to use visit this one:
```
https://github.com/tldr-pages/tldr/pull/5848/commits/29d67af0cf5077d9f0227818459d8de985469f89
```

**Comment by patricedenis** [comment]
All right.
I'm leaving now.
But I found a nice link in your above comment.
Don't touch anything, I will review then when I'm back.
Cheers

**Comment by patricedenis** [comment]
I just don't understand.
I used your link : [https://github.com/tldr-pages/tldr/pull/5848/commits/29d67af0cf5077d9f0227818459d8de985469f89](https://github.com/tldr-pages/tldr/pull/5848/commits/29d67af0cf5077d9f0227818459d8de985469f89)
but they are only two pages.fr files to review in this commit : 
image

Where am I wrong?

**Comment by bl-ue** [comment]
You're not wrong.  That's what I originally meant.  I merged `main` into this PR, but there were 4 conflicts, and two of them were french, and so I merged them as best I could, but I might have done it wrong.

GitHub/Refined GitHub _modified_ my link (apparently its a [known bug](https://github.com/sindresorhus/refined-github/issues/2327)) and so the one you clicked had way more than I wanted you to review...

That's where the confusion comes from.

**Comment by patricedenis** [comment]
Ok so that what I understand now:

- you merged main into one of your personal branches.
- When I first looked into the given [commit](https://github.com/tldr-pages/tldr/commit/29d67af0cf5077d9f0227818459d8de985469f89), I found several other pages.fr files where I should fix typos.
- If I want to fix these later, I should search into main?

**Comment by bl-ue** [comment]
- Correct
- Correct
- Yes, that would be nice.

**Comment by patricedenis** [comment]
Great OK I get it.
Thanks @bl-ue.
So I committed a little typo fix.
Now I can give you my approval for the French files but I didn't looked at the other files so I don't know if the approval would be correct as I reviewed a very tiny part of the PR. What is the usage in that king of use cases?

**Comment by bl-ue** [comment]
Well, this PR is actually about the English pages, so the French pages don't matter much right now.  The best thing to do would be for you to open a PR, fixing all of the French typos that you can.  I would myself, but I don't know French so I'd probably end up adding more typos than removing them 😆

**Comment by patricedenis** [comment]
Arf, you're right :smiley_cat: 
You may then remove my french commit then ^^
Ok, I will try to propose a PR soon for that. Cheers !

**Comment by CleanMachine1** [comment]
stash the commit @patricedenis and keep it for later!

**Comment by bl-ue** [comment]
>You may then remove my french commit then ^^

Why not just keep it?  I think I will keep it.

>stash the commit @patricedenis and keep it for later!

@CleanMachine1 you can't actually stash commits.  You can only stash uncommitted changes to tracked files.

**Comment by CleanMachine1** [comment]
Interesting, did try to learn how to use stash a while back and had some success, I thought you stashed unpushed commits

**Comment by bl-ue** [comment]
Stashing is invaluable.  I use it _all_ the time 👍🏻

File: pages.zh/osx/port.md (hunk 3 above)
Comment (sbrl) [reply pages.zh/osx/port.md:3]: This was an unusual, but very deliberate design choice by Apple to call their operating system macOS. It doesn't make proper grammatical sense, but it's the correct nonetheless.

File: pages/common/aria2.md (hunk 5 above)
Comment (sbrl) [reply pages/common/aria2.md:3]: It really depends on context as to which one I prefer. In this particular instance, I prefer it with a dash - but that is not transferable to other cases, which I would need to individually review.

**Review by sbrl — APPROVED** [review]
Thanks, @bl-ue!

On the subject of stashing, I've never actually found a use for it lol

I find it difficult to keep track of :woman_shrugging:

**Comment by patricedenis** [comment]
> > You may then remove my french commit then ^^
> 
> Why not just keep it? I think I will keep it.

I was just suggesting to remove it if we wanted to focus exclusively on English. But if some French is included in the Shakespeare's commit, that doesn't matter really much of course :tongue:

**Comment by CleanMachine1** [comment]
Will review but slowly,

**Comment by CleanMachine1** [comment]
Please tell me when you are done commiting

**Comment by bl-ue** [comment]
I'm done now 👍🏻

**Comment by bl-ue** [comment]
>Will review but slowly,

No problem.  Take your time.

File: pages/common/subfinder.md [hunk 11]
```diff
@@ -12,7 +12,7 @@
 
 `subfinder --silent -d {{example.com}}`
 
-- Use bruteforcing to find subdomains:
+- Use brute force to find subdomains:
```
Comment (marchersimon) [inline pages/common/subfinder.md:15]: I'm not sure if `brute force` would be taken a bit too literal in this context. (Altough it sounds really funny 😄). How about:
```suggestion
- Use a brute-force attack to find subdomains:
```

File: pages/linux/dnsrecon.md [hunk 12]
```diff
@@ -11,7 +11,7 @@
 
 `dnsrecon --domain {{example.com}} --name_server {{nameserver.example.com}} --type axfr`
 
-- Scan a domain, using a dictionary of subdomains and hostnames for bruteforcing:
+- Scan a domain, using brute force and a dictionary of subdomains and hostnames:
```
Comment (marchersimon) [inline pages/linux/dnsrecon.md:14]: Same as above.

File: pages.it/common/calibre-server.md [hunk 13]
```diff
@@ -1,11 +1,11 @@
 # calibre-server
 
-> Un'applicazione server che può essere usata per distribuire ebook in una rete.
-> Gli ebook devono prima essere importati nella libreria usando la GUI o calibredb.
-> Parte del manager di ebook Calibre.
+> Un'applicazione server che può essere usata per distribuire eBook in una rete.
+> Gli eBook devono prima essere importati nella libreria usando la GUI o calibredb.
```
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.it/common/calibre-server.md:4]: Darn, I should've checked.  I'll fix this all over.

File: pages/linux/dnsrecon.md (hunk 12 above)
Comment (bl-ue) [reply pages/linux/dnsrecon.md:14]:
```suggestion
- Scan a domain, using a brute-force attack and a dictionary of subdomains and hostnames:
```

File: pages/common/indent.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/indent.md:14]: Believe it or not, he had some hand in the birth of Golang, although I forget what (see [here](https://github.com/golang/go/commits/master?after=a2c07a9a1a445c28fa4db9bd1472b28cad3ca818+48070&branch=master))

**Comment by CleanMachine1** [comment]
Are you going to rebase this?

**Review by CleanMachine1 — APPROVED** [review]

**Comment by bl-ue** [comment]
If you mean rebase against `main` (which we do to update the PR with the latest changes), the answer is no.  I merged `main` in a few hours ago.

If you mean rebase when merging instead of squashing (which I think you do), the answer is also no.  The commits period weren't _that_ clean, but I added a bunch of messiness which would be great to keep **out** of `main`.

I'd definitely like a more approvals before merging though, as this is a large PR (modifying 190 files in one go isn't an everyday affair.)

**Comment by bl-ue** [comment]
This batch of typos, along with a few others that I found for fun last night, are shipped for admittance into codespell @ https://github.com/codespell-project/codespell/pull/1919 🎉

File: pages.it/common/calibre-server.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.it/common/calibre-server.md]: → 4daeb1ee1bf619c058101227c8c38f51a4fcbdf2

**Review by marchersimon — APPROVED** [review]
Great work, thanks @bl-ue

**Review by patricedenis — APPROVED** [review]
As I said I reviewed the English and French pages, but I'm not able to check for other languages.
By the way, that's a huge work. Congrats to you @bl-ue !!

**Comment by bl-ue** [comment]
@navarroaxel, @Waples & @nicokosi will you please approve? Then I'll merge.

**Review by SethFalco — COMMENTED** [review]
Just a few subjective comments!

The "deduplicate" one might be irrelevant, it's a British English thing I think.

**Comment by SethFalco** [comment]
I'm also looking at this through [LanguageTool](https://languagetool.org/) to see if that can spot anything.

Strongly-named -> Strongly named

It picks up these, and I'd agree even if "uncompress" is valid, "decompress" is more notable, might be worth switching:
uncompressing -> decompressing
uncompress -> decompress

In many cases, but this change can't be done in bulk blindly:
it's -> its

"Its" shouldn't have the apostrophe when referring to something that belongs to "it".

I'll continue browsing it in the evening, it could be what LanguageTool reports could be done in a separate PR.

File: pages/common/borg.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # borg
 
-> Deduplicating backup tool.
+> De-duplicating backup tool.
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/borg.md:3]: Another great catch. The hyphen `-` is totally unnecessary here.

File: pages/common/hr.md [hunk 15]
```diff
@@ -10,6 +10,6 @@
 
 `hr {{string}}`
 
-- Print a multiline horizontal rule:
+- Print a multi-line horizontal rule:
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/hr.md:13]: :+1:

This should be reversed too.

File: pages/common/siege.md [hunk 16]
```diff
@@ -3,19 +3,19 @@
...
 
 `siege --file {{path/to/url_list.txt}}`
 
-- Test list of urls in a random order (Simulates internet traffic):
+- Test list of URLs in a random order (Simulates internet traffic):
 
 `siege --internet --file {{path/to/url_list.txt}}`
 
-- Benchmark a list of urls (Dont wait between requests):
+- Benchmark a list of URLs (Don't wait between requests):
```
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/common/siege.md:18]: I like `Doesn't` best here.

File: pages/common/borg.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/borg.md:3]: Apparently it's an American word, too, although I didn't realize.  I'll change it 👍🏻

File: pages/common/hr.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/hr.md:13]: I didn't even think to check before I began hyphenizing 😖

File: pages/common/siege.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/siege.md:18]: I'm in favor of the latter, `without waiting`.

File: pages/common/unzip.md [hunk 17]
```diff
@@ -18,6 +18,6 @@
 
 `unzip -c {{file.zip}}`
 
-- Extract a zip file created in windows, containing files with non-ascii (chinese) filenames:
+- Extract a zip file created on Windows, containing files with non-ASCII (e.g. Chinese or Japanese character) filenames:
```
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/unzip.md:21]: Of course, my mistake.

File: pages.th/common/tldr.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # tldr
 
-> แสดงตัวอย่างแบบง่ายสำหรับเครื่องมือบน command-line จากโปรเจคท์ tldr-pages.
+> แสดงตัวอย่างแบบง่ายสำหรับเครื่องมือบน command line จากโปรเจคท์ tldr-pages.
```
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages.th/common/tldr.md:3]: We discussed this [here](https://github.com/tldr-pages/tldr/pull/5848#discussion_r624893662) but came to no conclusion.  I personally prefer the hypen, so I'm glad it's the right one.  I'll fix it.

File: pages/common/siege.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/siege.md:18]:
```suggestion
- Benchmark a list of URLs (without waiting between requests):
```

**Comment by Waples** [comment]
I'm approving and we'll see @SethFalco additions in the new PR

**Review by Waples — APPROVED** [review]

**Review by nicokosi — APPROVED** [review]
👌 for me

**Review by patricedenis — APPROVED** [review]

**Review by SethFalco — APPROVED** [review]
Gets my seal of approval! \\(^-^)/

**Review by sbrl — APPROVED** [review]
Double-digit approvals! I think this is about ready to merge.

I think this is also a record number of approvals too.

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
9 approvals! 
Did someone say that we should wait, or can we push this PR?

**Comment by bl-ue** [comment]
:shipit:
